packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -1235,6 +1237,208 @@ describe('RpcContractProvider test', () => {
     });
   });
 
+  describe("transferTicket", () => {
+    it("validate that a reveal operation will be added when needed", async (done) => {
+      mockRpcClient.getBlockMetadata.mockResolvedValue(TransferTicketNoReveal)
```

Comment:
Wondering if this line is required, `TransferTicketNoReveal` is doesn't match the format of response for the getBlockMetadata entrypoint